package/yast2-users.spec

```diff
@@ -64,6 +66,9 @@ Requires:       yast2-core >= 2.21.0
 
 Requires:       yast2-ruby-bindings >= 1.0.0
 
+# ssh-keygen
+Requires:       openssh
```

Comment:
SSH is a quite basic tool so it will be very likely present even in a small system. But do we need it also in installed system or is this feature present only in installation?

Reply:
It is true that we are targeting the installation only... by now. But as part of this feature, we will provide a way to manage public keys to users in an already installed system and we might need this tool.